tests/integration/test_hyper_transformer.py

```diff
@@ -228,7 +228,7 @@ def test_dtype_category():
 
     rever = ht.reverse_transform(trans)
 
-    pd.testing.assert_frame_equal(df, rever)
```

Comment:
What is this change for? Also, I know it's not your code but personally I prefer to write out the full names unless it's something super long. You can leave that though

Reply:
Oh, I just forgot to revert back. I agree that some of the tests need to be standardized.